.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,16 @@
+name: Release Drafter
```

Comment:
Actually... we've got both CircleCI and GH actions now... let's make it consistent

Reply:
AFAICS there's quite a lot of ready logic in these 2 actions that might not be readily available in CircleCI... so maybe the existing CircleCI pipeline (which is rather trivial) could be migrated to a GH action

Reply:
Oh okay, mostly `release-drafter` since the other one has just `setup-scala`

Reply:
opened #84